pkg/backup/compaction_job.go

```diff
@@ -1016,6 +767,7 @@ func processProgress(
 			manifest.Files = append(manifest.Files, file)
 			manifest.EntryCounts.Add(file.EntryCounts)
 		}
+		// TODO (kev-cao): Add per node progress updates.
```

Comment:
what is this referring to? i think we just need to persist the intermediate manifests, right?

Reply:
I noticed that `backup_job` has [progress updates for each node](https://github.com/kev-cao/cockroach/blob/673c8c1796855bf2f729a1f9106a24217713d3d2/pkg/backup/backup_job.go#L231-L244), that seemed like a nice to have for compactions too.